bundle/permissions/check.go

```diff
@@ -0,0 +1,99 @@
+package permissions
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+)
+
+type WorkspacePathPermissions struct {
+	Path        string
+	Permissions []resources.Permission
+}
+
+func ObjectAclToResourcePermissions(path string, acl []workspace.WorkspaceObjectAccessControlResponse) *WorkspacePathPermissions {
+	permissions := make([]resources.Permission, 0)
+	for _, a := range acl {
+		// Skip the admin group because it's added to all resources by default.
+		if a.GroupName == "admin" {
+			continue
+		}
```

Comment:
This doesn't work.

I got the following on a test deploy:
```
Warning: permissions missing

The following permissions apply to the workspace folder at "/Workspace/Users/pieter.noordhuis@databricks.com/.bundle/my_permission_validation_check/prod" but are not configured in the bundle:
- level: CAN_MANAGE, group_name: admins
```

Reply:
Ooph, good catch!